<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInfoDialog</class>
 <widget class="QDialog" name="FileInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>288</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="fileInfoGrpBox">
     <property name="title">
      <string>File Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="titleLbl">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Region</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="ntscURadioBtn">
           <property name="text">
            <string>NTSC-U</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">regionBtnGrp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ntscJRadioBtn">
           <property name="text">
            <string>NTSC-J</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">regionBtnGrp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="palRadioBtn">
           <property name="text">
            <string>PAL</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">regionBtnGrp</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="bannerImg">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>192</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;NO BANNER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="adventureCountLbl">
        <property name="text">
         <string>Adventure Count:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="checkSumLbl">
        <property name="text">
         <string>Checksum:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="currentAdventureLbl">
        <property name="text">
         <string>Current Adventure:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="subtitleLbl">
        <property name="text">
         <string>Subtitle:</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>checkSumLbl</zorder>
     <zorder>adventureCountLbl</zorder>
     <zorder>currentAdventureLbl</zorder>
     <zorder>groupBox</zorder>
     <zorder>titleLbl</zorder>
     <zorder>subtitleLbl</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ntscURadioBtn</tabstop>
  <tabstop>ntscJRadioBtn</tabstop>
  <tabstop>palRadioBtn</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="regionBtnGrp"/>
 </buttongroups>
</ui>
